repos/system_upgrade/common/actors/scanpkgmanager/libraries/scanpkgmanager.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from leapp.libraries.stdlib import api
+from leapp.models import PkgManagerInfo
+from leapp.libraries.common.config.version import get_source_major_version
+
+
+def _get_releasever_path():
+    default_manager = 'yum' if get_source_major_version() == '7' else 'dnf'
+    return '/etc/{}/vars/releasever'.format(default_manager)
+
+
+def _releasever_exists(releasever_path):
+    return bool(os.path.isfile(releasever_path))
```

Comment:
The conversion to bool is not necessary as function returns bool.
```suggestion
    return os.path.isfile(releasever_path)
```

Reply:
Ouch :D This happens when you are just blindly following linter...